src/App.ts

```diff
@@ -73,6 +73,7 @@ export interface AppOptions {
   logLevel?: LogLevel;
   ignoreSelf?: boolean;
   clientOptions?: Pick<WebClientOptions, 'slackApiUrl'>;
+  processBeforeResponse?: boolean;
```

Comment:
might be better to illustrate the relationship of this type to `ExpressReceiver` by putting at the top (with the other options that come from `ExpressReceiverOptions`) and by simply referring to the property within that type.